File: src/main/java/me/rasztabiga/fridgy/products/repository/RecipeIngredientRepository.java
package me.rasztabiga.fridgy.products.repository;

import me.rasztabiga.fridgy.products.domain.RecipeIngredient;
import org.springframework.data.jpa.repository.*;
import org.springframework.stereotype.Repository;

/**
 * Spring Data SQL repository for the RecipeIngredient entity.
 */
@SuppressWarnings("unused")
@Repository
public interface RecipeIngredientRepository extends JpaRepository<RecipeIngredient, Long> {}

File: src/main/kotlin/me/rasztabiga/fridgy/products/repository/RecipeIngredientRepository.kt
package me.rasztabiga.fridgy.products.repository

import me.rasztabiga.fridgy.products.domain.RecipeIngredient
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

/**
 * Spring Data SQL repository for the [RecipeIngredient] entity.
 */
@Suppress("unused")
@Repository
interface RecipeIngredientRepository : JpaRepository<RecipeIngredient, Long>
